src/ngraph/runtime/interpreter/int_backend.cpp

```diff
@@ -202,20 +220,7 @@ bool runtime::interpreter::INTBackend::call(shared_ptr<Function> function,
         }
         if (instance.m_nan_check_enabled)
         {
-            perform_nan_check(op_outputs, op);
-        }
-
-        // delete any obsolete tensors
-        for (const descriptor::Tensor* t : op->liveness_free_list)
```

Comment:
Should we not still do this cleanup one way or another?

Reply:
We never allocate temporaries except for the single AlignedBuffer which is deleted on backend destruction